controller_interface/src/controller_interface_base.cpp

```diff
@@ -123,8 +125,8 @@ const rclcpp_lifecycle::State & ControllerInterfaceBase::configure()
   }
   if (is_async_)
   {
-    const unsigned int thread_priority =
-      static_cast<unsigned int>(get_node()->get_parameter("thread_priority").as_int());
+    const int thread_priority =
+      static_cast<int>(get_node()->get_parameter("thread_priority").as_int());
```

Comment:
If we get as int, do we still need static_cast here?

Reply:
don't think so, I changed it

Reply:
```
  /home/runner/work/ros2_control/ros2_control/.work/target_ws/src/ros2_control/controller_interface/src/controller_interface_base.cpp:129:51: error: implicit conversion loses integer precision: 'int64_t' (aka 'long') to 'const int' [-Werror,-Wshorten-64-to-32]
    128 |     const int thread_priority =
        |               ~~~~~~~~~~~~~~~
    129 |      get_node()->get_parameter("thread_priority").as_int();
```

Reply:
Got it. Thanks